README.md

```diff
@@ -3,7 +3,7 @@
 > An AWS CloudFormation stack to run a serverless password-protected photo
   gallery
 
-**Demo:** <https://awspics.net>  
+**Demo:** <https://awspics.net>
```

Comment:
These trailing spaces were intentional to have Credentials on a new line.

Reply:
Shouldn't rely on trailing spaces for this, or for anything else - trailing spaces are discouraged, and many people (including but not limited to me!) have their editors set to strip all trailing spaces when editing files.

I've added another newline between Demo and Credentials, and from a quick test, that seems to make GitHub render them on separate lines.

Reply:
I didn't _make_ the Markdown spec, I just _write_ Markdown 😆 

Your latest change isn't equivalent to what was there before, but I don't feel strongly:

![image](https://user-images.githubusercontent.com/474794/98860111-082e5000-2418-11eb-9501-1a60b095bf8c.png)

Reply:
Oh yeah, true ... sorry about that.

Seems that GitHub renders Markdown slightly differently to other renderers, too (although I already vaguely knew that). For example, if I paste the text in to dillinger.io with neither the trailing spaces nor the extra newline, it renders Demo and Credentials on separate lines.

![readme-in-dillinger](https://user-images.githubusercontent.com/79373/98862012-4e3bf000-24ba-11eb-997c-bd30dda65490.png)

Reply:
No problem at all, just sharing why I went with the trailing spaces approach originally.